dev-docs/bidders/eskimi.md

```diff
@@ -6,19 +6,24 @@ pbjs: true
 pbs: false
 biddercode: eskimi
 deals_supported: false
-media_types: banner
+media_types: banner, video
```

Comment:
If you support multiple formats we highly  encourage you to add the following meta data as well

```
multiformat_supported: will-bid-on-any, will-bid-on-one, will-not-bid
```

Reply:
👍☺️